stories/submission.stories.js

```diff
@@ -0,0 +1,118 @@
+// https://github.com/EnCiv/undebate-ssp/issues/51
+import React from 'react'
+import Submission from '../app/components/submission'
+
+export default {
+    title: 'Submission',
+    component: Submission,
+    argTypes: {},
+}
+
+const Template = (args, context) => {
+    const { onDone } = context
+    return <Submission electionObj={onDone} {...args} />
+}
+
```

Comment:
Please add a test for Empty, where electionObj={}.  This will be what happens when the admin first creates a blank election.